AppDB/cassandra_utils.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+#
+# Simple script to rebalance the load on a Cassandra pool. This works only for
+# AppScale deployment.
+#
+# Author: graziano
+
+# Cassandra nodetool.
+CMD="/root/appscale/AppDB/cassandra/cassandra/bin/nodetool"
+
+# The keyspace to work on.
+KEYSPACE="Keyspace1"
+
+set -e
+
```

Comment:
There should be a check like
[ -x "$CMD" ] || { echo "Cannot find nodetool command!"; exit 1 }
to ensure CMD exists.